crates/cli/src/subcommands/generate/csharp.rs

```diff
@@ -647,7 +652,7 @@ pub fn autogen_csharp_globals(items: &[GenItem], namespace: &str) -> Vec<(String
         );
         writeln!(
             output,
-            "public ReducerEvent(ClientApi.Event dbEvent, IReducerArgs? args) : base(dbEvent) => Args = args;"
+            "public ReducerEvent(TransactionUpdate update, IReducerArgs? args) : base(update) => Args = args;"
```

Comment:
Either we need to add `using SpacetimeDB.ClientApi;` to the top of this file or you need to fully qualify this path because it causes a compilation issue:

```
Assets/_Project/autogen/_Globals/SpacetimeDBClient.cs(515,23): error CS0246: The type or namespace name 'TransactionUpdate' could not be found (are you missing a using directive or an assembly reference?)
```

Reply:
I believe this still doesn't compile. When I run the autogen for the BitCraft module I still end up with this compilation error:

<img width="961" alt="image" src="https://github.com/clockworklabs/SpacetimeDB/assets/4099508/d93243e9-8015-4eb7-a57a-a5136bd62bdb">

```
Assets/_Project/autogen/_Globals/SpacetimeDBClient.cs(524,23): error CS0246: The type or namespace name 'TransactionUpdate' could not be found (are you missing a using directive or an assembly reference?)
```